### PR TITLE
feat: infinite scroll 20% paging and bigger range

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Because we fetch certified data, it takes a certain time to get and render the data. That's why to improve the experience it was decided to fetch 100 elements instead of 10 and to modify the infinite scroll to not observe the last element but rather an element that finds place at 20% of the last displayed data.

# Changes

- bump list constant form 10 to 100
- modify `InfiniteScroll` to observe an element that finds place at 20% instead of `lastElement`

# Tests

As previous PR regarding infinite scroll, this needs e2e testing to improve tests.

### CODE REVIEW

File: frontend/svelte/src/lib/components/ui/InfiniteScroll.svelte [hunk 1]
```diff
@@ -48,11 +52,50 @@
...
+    if (!container.children.length) {
       return;
     }
 
-    observer.observe(container.lastElementChild);
+    const pageIndex: number =
+      container.children.length / LIST_PAGINATION_LIMIT - 1;
+
+    // If the pageIndex is not an integer the all page was not fetched - e.g. 50 elements instead of 100 - therefore there is no more elements to fetch
+    if (!Number.isInteger(pageIndex)) {
```
Comment (mstrasinskis) [inline frontend/svelte/src/lib/components/ui/InfiniteScroll.svelte:63]: It relies on rendered element count. Will this work with "Open" filter hack?

File: frontend/svelte/src/tests/lib/components/ui/InfiniteScroll.spec.ts [hunk 2]
```diff
@@ -27,9 +28,22 @@ describe("InfiniteScroll", () => {
...
 
     expect(spyIntersect).toHaveBeenCalled();
   });
+
+  it("should not trigger an intersect event", () => {
+    const spyIntersect = jest.fn();
+
+    render(InfiniteScrollTest, {
+      props: {
+        elements: new Array(LIST_PAGINATION_LIMIT + 1),
```
Comment (mstrasinskis) [inline frontend/svelte/src/tests/lib/components/ui/InfiniteScroll.spec.ts:42]: `new Array(LIST_PAGINATION_LIMIT - Math.round(LIST_PAGINATION_LIMIT * INFINITE_SCROLL_OFFSET) + 1)`?

**Review by mstrasinskis — COMMENTED** [review]
Nice idea! Just a couple of questions.

File: frontend/svelte/src/lib/components/ui/InfiniteScroll.svelte (hunk 1 above)
Comment (peterpeterparker) [reply frontend/svelte/src/lib/components/ui/InfiniteScroll.svelte:63]: Yes. The "open" filter hack hide the proposal content (the card) but still render empty divs in the DOM.

File: frontend/svelte/src/tests/lib/components/ui/InfiniteScroll.spec.ts (hunk 2 above)
Comment (peterpeterparker) [reply frontend/svelte/src/tests/lib/components/ui/InfiniteScroll.spec.ts:45]: Thanks. Just added this test too.

File: frontend/svelte/src/lib/components/ui/InfiniteScroll.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/svelte/src/lib/components/ui/InfiniteScroll.svelte:63]: Ah, true. Thanks for description.

**Review by mstrasinskis — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**